python/ray/scripts/scripts.py

```diff
@@ -664,6 +683,19 @@ def get_head_ip(cluster_config_file, cluster_name):
     click.echo(get_head_node_ip(cluster_config_file, cluster_name))
 
 
+@cli.command()
+@click.argument("cluster_config_file", required=True, type=str)
+@click.option(
+    "--cluster-name",
+    "-n",
+    required=False,
+    type=str,
+    help="Override the configured cluster name.")
+def get_worker_ips(cluster_config_file, cluster_name):
+    worker_ips = get_worker_node_ips(cluster_config_file, cluster_name)
+    click.echo('\n'.join(worker_ips))
```

Comment:
```suggestion
    click.echo("\n".join(worker_ips))
```

Reply:
I see; I guess it's not that big of a deal - no convention anywhere, but iirc I think most of the python code doesn't have single quotes.

Reply:
We've been sticking to double quotes, but we haven't written it down anywhere.